.github/workflows/build_docker_image.yml

```diff
@@ -15,8 +15,6 @@ jobs:
     runs-on: ubuntu-latest
     env: 
       tag_name: ${{ github.event.inputs.docker_image_tag_name || github.ref_name }}
-    outputs:
```

Comment:
dont we need this?

Reply:
he was asking to put the `needs.push_to_registry.outputs.image_name` to env and use it from there.